charms/trusty/clearwater-aio-proxy/README.md

```diff
@@ -0,0 +1,49 @@
+# Overview
+
+This is a [Juju charm](https://jujucharms.com/about), which allows configuration of the [Project Clearwater](http://projectclearwater.org) IMS core's [all-in-one](http://clearwater.readthedocs.org/en/stable/All_in_one_Images/index.html) node.
+
+This is a proxy charm, meaning that you must spin up the all-in-one VM first, and then point this charm at it to manage it.
+
+Since the all-in-one node does not support scaling up, neither does this charm.
+
+# Deployment
+
+## Initial deployment
+
+The all-in-one VM image should be downloaded from [http://repo.cw-ngv.com/juju-clearwater-2/cw-aio.ova](http://repo.cw-ngv.com/juju-clearwater-2/cw-aio.ova) and deployed onto your virtualization platform.  (You could alternatively try the latest all-in-one VM image from [http://vm-images.cw-ngv.com/](http://vm-images.cw-ngv.com/), but this may not have been tested with this charm - the juju-clearwater-2 version above is known to work.)
+
+The proxy charm should then be deployed, pointing at the all-in-one VM.
+
+# Using the All-in-One Node
+
+Once installed, the all-in-one node will listen for SIP traffic on port 5060 (both TCP and UDP).  You can use a standard SIP client (e.g. Blink, Boghe or X-Lite) to register against Bono's public IP (which you can find with `juju status clearwater-bono`) and make calls.
+
+Our ["Making your first call" documentation](http://clearwater.readthedocs.org/en/latest/Making_your_first_call/index.html) has more information on this process.
+
+# Configuration
+
+-  `proxied_ip`: The IP address of the All-in-One node to manage
+-  `home_domain`: The home domain for this service
+-  `base_number`: The first number to be allocated in the number range
+-  `number_count`: The count of numbers to allocate
+
+# Actions
```

Comment:
I think these probably have no relation to `base_number` and `number-count`, which control Ellis function - worth making that explicit?